.golangci.yml

```diff
@@ -18,6 +18,9 @@ linters:
 linters-settings:
   goimports:
     local-prefixes: k8s.io/kube-state-metrics
+  gocyclo:
```

Comment:
I will open an issue for this after the PR is merged, to refactor the podMetricFamilies function, note this function did not change, it was just a var before.